template-bff-service/package.json

```diff
@@ -27,14 +27,16 @@
       "command": "nyc mocha --recursive \"./test/unit/**/*.test.js\"",
       "env": {
         "NODE_ENV": "test",
+        "AES": "false",
         "AWS_REGION": "us-west-2",
         "KMS_REGIONS": "us-east-1,us-west-2"
       }
     },
     "test:int": {
-      "command": "npm start -- --exec \"mocha --timeout 20000 --require @babel/register --recursive ./test/helper.js \"./test/int/**/*.test.js\"\"",
+      "command": "start-test --expect 404 http://localhost:3002 'mocha --timeout 20000 --require @babel/register --recursive ./test/helper.js \"./test/int/**/*.test.js\"'",
```

Comment:
serverless-offline no longer supports the --exec arg

Reply:
- the lambda endpoint of 3002 returns a 404 when start-test sends a ping request
- since there is no ping function defined
- but that is good enough to know that serverless-offline is up